mlir/lib/Analysis/AliasAnalysis/LocalAliasAnalysis.cpp

```diff
@@ -40,17 +40,19 @@ using namespace mlir;
 static constexpr unsigned maxUnderlyingValueSearchDepth = 10;
 
 /// Given a value, collect all of the underlying values being addressed.
```

Comment:
Can you add documentation for the arguments? Here and for all functions who's signature has changed.